website/views.py

```diff
@@ -241,6 +242,10 @@ def resolve_guid(guid, suffix=None):
         if not referent.deep_url:
             raise HTTPError(http.NOT_FOUND)
         if isinstance(referent, PreprintService):
+            if re.match('download/?', suffix) and referent.primary_file:
```

Comment:
I'm not certain about the beginning and ending slashes - regex seems to be overkill... and is slower

`suffix.starts_with('/download')`